### ISSUE TITLE
Support Kubernetes 1.20

### ISSUE THREAD

**Issue by neweyes** [superedge/superedge#72]
**What would you like to be added/modified**:
application-grid-wrapper/application-grid-controller/statefulset-grid-daemon should be modified

**Why is this needed**:
Kubernetes 1.20 enable [EndpointSlices](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#endpointslices-are-now-enabled-by-default) by default，this feature can't work well with SuperEdge now.

**Comment by zentavr** [comment]
seems like it works with SuperEdge 0.7.0...?

**Comment by dodiadodia** [comment]
Yes, we have supported K8s 1.20 version and EndpointSlice since 0.7.0
